srcpkgs/libsignal-protocol-c/template

```diff
@@ -16,3 +16,13 @@ checksum=c22e7690546e24d46210ca92dd808f17c3102e1344cd2f9a370136a96d22319d
 if [ "${XBPS_CHECK_PKGS}" ]; then
 	configure_args+=" -DBUILD_TESTING=1"
 fi
+
+libsignal-protocol-c-devel_package() {
+	short_desc+=" - development files"
+	depends="${sourcepkg}>=${version}_${revision}"
+	pkg_install() {
+		vmove usr/include
+		vmove usr/lib/pkgconfig
+		vmove "usr/lib/*.so"
```

Comment:
Does it stop creating the static library once you enable shared libs?

Reply:
Yes, `-DBUILD_SHARED_LIBS=ON` stops creating the .a lib.